openstack/keppel/values.yaml

```
@@ -118,9 +118,13 @@ postgresql:
     enabled: true
     accessMode: ReadWriteOnce
     size: 100Gi
+  postgresDatabase: keppel
   sharedMemoryLimit: 256Mi
+  users:
+    keppel:
```

Maybe just a matter of personal taste, but having an implicit null value makes it looks like you forgot to type something.
```suggestion
  users:
    keppel: {}
```